src/app/core/services/search-criteria/search-criteria.service.ts

```diff
@@ -57,6 +59,7 @@ const URL_PARAM_ORDER_BY = 'orderBy';
 const URL_PARAM_REGION = 'regions';
 const URL_PARAM_ARRAY_DELIMITER = '~'; //https://www.rfc-editor.org/rfc/rfc3986#section-2.3
 const VALID_GEO_HAZARDS = new Set([[60, 20], [70], [10]]);
+const ULR_COORDS_PRECISION = 8;
```

Comment:
Hvorfor trenger du så mange desimaler i URL'en?
Er vi ikke nede på millimeter-presisjon da?
Jeg skjønner at det blir enklere å sjekke om bounds har endret seg, men jeg synes kanskje det er verdt å legge inn en feilmargin i sammenligninga i stedet for å ha 16 ekstra tegn i URL. Opplys meg gjerne om jeg ikke har skjønt det!